bash_completion.py

```diff
@@ -73,26 +76,30 @@ def _bash_completion_paths_default():
     the current platform.
     """
     platform_sys = platform.system()
-    if platform_sys == 'Linux' or sys.platform == 'cygwin':
-        bcd = ('/usr/share/bash-completion/bash_completion', )
-    elif platform_sys == 'Darwin':
-        bcd = ('/usr/local/share/bash-completion/bash_completion',  # v2.x
-               '/usr/local/etc/bash_completion')  # v1.x
-    elif platform_sys == 'Windows':
+    if platform_sys == "Linux" or sys.platform == "cygwin":
+        bcd = ("/usr/share/bash-completion/bash_completion",)
+    elif platform_sys == "Darwin":
+        bcd = (
+            "/usr/local/share/bash-completion/bash_completion",  # v2.x
+            "/usr/local/etc/bash_completion",
+        )  # v1.x
+    elif platform_sys == "Windows":
         gfwp = _git_for_windows_path()
         if gfwp:
-            bcd = (os.path.join(gfwp, 'usr\\share\\bash-completion\\'
-                                      'bash_completion'),
-                   os.path.join(gfwp, 'mingw64\\share\\git\\completion\\'
-                                      'git-completion.bash'))
+            bcd = (
+                os.path.join(gfwp, "usr\\share\\bash-completion\\" "bash_completion"),
+                os.path.join(
+                    gfwp, "mingw64\\share\\git\\completion\\" "git-completion.bash"
+                ),
+            )
         else:
             bcd = ()
     else:
         bcd = ()
     return bcd
 
 
-_BASH_COMPLETIONS_PATHS_DEFAULT = None
+_BASH_COMPLETIONS_PATHS_DEFAULT: tp.Tuple[str, ...] = ()
```

Comment:
This change broke default usage; since `_get_bash_completions_source` checks if `_BASH_COMPLETIONS_PATHS_DEFAULT` is `None`, it never gets initialized, and all completion attempts return an empty set:
```bash
▸ python -m bash_completion "ls --s"
▸ python - "ls --s" <<EOF
import bash_completion
bash_completion._BASH_COMPLETIONS_PATHS_DEFAULT = None
bash_completion._bc_main()
EOF
ls --show-control-chars
ls --si
ls --size
ls --sort
ls --sort=
```

Reply:
fixed and released 0.2.8